<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,10">
    <item>
     <layout class="QVBoxLayout" name="layoutSelection">
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHistogram">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>histogram</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnInvert">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>invert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConvolution">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>convolution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHough">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Hough</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLaneDec">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>lane detection 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLaneDec2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>lane detection 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLaneDec3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>lane detection 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRailDec">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>rail detection</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layoutMain">
      <item>
       <widget class="QLabel" name="imageLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
